cdap-ui/app/cdap/api/caskmarket/index.js

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright © 2016 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+import Datasource from '../../services/datasource';
+import {apiCreatorAbsPath} from '../../services/resource-helper';
+
+let dataSrc = new Datasource();
+const basepath = 'http://marketplace.cask.co.s3.amazonaws.com';
```

Comment:
@nitinmotgi This ideally will come from cdap-site.xml. Will work with the backend people to add it to cdap-site.xml. 

Opened JIRA: https://issues.cask.co/browse/CDAP-7303